### ISSUE TITLE
Linux application starts too tall, hiding essential buttons and window decoration

### ISSUE THREAD

**Issue by PanderMusubi** [Gulden/gulden-official#163]
#### If so, please list the steps to reproduce below:
1. Install Gulden Linux 64 bit on Ubuntu
2. run gulden
3. choose restore wallet or new wallet

### Expected behaviour
The application should start in normal maximized size with window decoration visible and all buttons visible.

### Actual behaviour
The window is too tall for the screen. The window decoration is not visible and, more importantly, the button to continue to the next screen at the bottom (e.g.Recover wallet) is also not visible. The user does not know how to continue. It looks as if some buttons are missing or the app is stuck.

The workaround is to find the edge of the window and so resize or remaximize the application so the entire window is visible. I assume the width of the screen is also used for the height when the application starts.

Even for experienced users, it is very hard to find the workaround. This prevents users, starting with Gulden. Even though it is a small bug, it is a show stopper for many when encountered.

### Screenshots.
![2019-05-26-133608_1920x1080_scrot](https://user-images.githubusercontent.com/198964/58381273-85af9780-7fbb-11e9-8072-f2a458af6a46.png)

![2019-05-26-133709_1920x1080_scrot](https://user-images.githubusercontent.com/198964/58381267-80eae380-7fbb-11e9-8499-42edc773a529.png)

### What version of Gulden are you using?
https://github.com/Gulden/gulden-official/releases/download/v2.1.0.4/Gulden-2.1.0.4-x86_64.AppImage

### Machine specs:
- OS: Ubuntu (18.04/18.10/19.04) with LXQt 0.14.1 (this is not so much as fringe windows manager and comes default with Lubuntu (and directly is installable on Ubuntu itself too). Because it is very light, it is popular on older hardware or with users that don't want to have heavy window managers. It is more promising that Xubuntu)
- CPU: n/a
- RAM: n/a
- Disk size: n/a
- Disk Type (HD/SDD): n/a

**Comment by mjmacleod** [comment]
2.1.0.5 pre-release builds are out, kindly give them a try and let me know if anything changes regarding this.

**Comment by PanderMusubi** [comment]
Unfortunately, still an issue for 2.1.0.7

**Comment by mjmacleod** [comment]
At what resolution are you running? I think I can produce this now on the latest ubuntu but only when running at a very small resolution.

**Comment by PanderMusubi** [comment]
1920 x 1080

**Comment by PanderMusubi** [comment]
Did you run it with LXQt? (Lubuntu?)

**Comment by mjmacleod** [comment]
Probably not the same issue then...

Yes I installed the latest lubuntu  "lubuntu-19.04-desktop-amd64.iso" with completely default settings and immediately installed and ran the Gulden .AppImage 

I can't reproduce anything similar unless I set the resolution below 800x600 - I wish I could reproduce it because then it would likely be easy to fix. I tried previously (when bug was reported) with lubuntu 18 as well and similar situation.

I'll fix the 800x600 resolution issue anyway and see if that helps.
